CodeGen/Generators/NanoFrameworkGenerator.cs

```diff
@@ -126,85 +122,183 @@ public static void Generate(string rootDir, Quantity[] quantities, bool updateNa
             Log.Information("");
         }
 
-        private static void SetDependencyVersions(FindPackageByIdResource resource, SourceCacheContext cache, ILogger logger,
-            CancellationToken cancellationToken, bool updateNanoFrameworkDependencies, string outputDir)
+        /// <summary>
+        /// Updates existing nanoFramework projects and nuspecs with the latest versions.
+        /// </summary>
+        /// <param name="rootDir">The root directory</param>
+        /// <param name="quantities">The quantities to update nuspecs</param>
+        public static void UpdateNanoFrameworkDependencies(
+            string rootDir,
+            Quantity[] quantities)
         {
-            if (updateNanoFrameworkDependencies)
+            // working path
+            string path = Path.Combine(rootDir, "UnitsNet.NanoFramework\\GeneratedCode");
+
+            Log.Information("");
+            Log.Information("Updating .NET nanoFramework references using nuget CLI");
```

Comment:
I ran `generate-code.sh` locally, and it seems to downgrade from stable to preview package?

![image](https://user-images.githubusercontent.com/787816/129488722-e94ca590-3b14-49da-99aa-1f82c0d787e4.png)

Reply:
Seems related to Tools/NuGet.exe version being too old and `nuget update` failed silently in codegen.
I pushed some commits updating it from 4.9.3 to 5.10.0 and then it was able to update the nuget versions.

Reply:
See above, I couldn't push commits to this fork.

Reply:
Got you! Merged.